zstacklib/zstacklib/utils/gpu.py

```
@@ -231,6 +232,24 @@ def reload_hygon_gpu_driver_cmd():
     return cmd
 
 
+def get_vastai_type():
+    r, o, e = bash_roe("lspci | grep -E 'Vastai|1ec6'")
```

Ruff (0.12.2)

`bash_roe` may be undefined, or defined from star imports

(F405)